client/me/billing-history/receipt.jsx

```diff
@@ -126,6 +130,36 @@ const BillingReceipt = React.createClass( {
 		);
 	},
 
+	renderTitle() {
+		const { translate } = this.props;
+
+		return (
+			<div>
+				<QueryBillingTransactions />
```

Comment:
We'll always want to render the query component on mount, and make data needs as clear as possible. How about something similar to:
```javascript
render() {
    const { transaction } = this.props;
    return(
        <Main>
            <QueryBillingTransactions />
            { this.renderTitle() }
            { ! transaction ? this.renderPlaceholder() : this.renderBillingHistory() }
       </Main>
   );
}
```